.github/workflows/test.yaml

```diff
@@ -33,7 +33,12 @@ jobs:
     container: ${{ matrix.image }}
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Setup environment
```

Comment:
Can you add a comment here why this occurs and linking to https://github.com/canonical/snapd-glib/issues/168. Hopefully we can remove this at some point.

Reply:
AFAIU, nodejs 20 requires a newer GLib version than the one available in 18.04, so this patch specifies to not use nodejs20 and allow to use an insecure, old version.